src/BenchmarkDotNet/ListBenchmarks/AsciiTreeDiagram.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+
+namespace BenchmarkDotNet.ListBenchmarks {
```

Comment:
could you please move all of the types from `BenchmarkDotNet.ListBenchmarks` to `BenchmarkDotNet.ConsoleArguments.ListBenchmarks`? BDN project has already too many folders, we try to avoid adding new ones on the root level